modules/op-tkn-hub-interaction.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// *  cli_reference/tkn_cli/op-tkn-reference.adoc
+
+[id="op-tkn-hub-interaction_{context}"]
+= Hub interaction commands
+
+Interact with Tekton Hub for resources such as tasks and pipelines.
+
+== hub
+Interact with hub.
+
+.Example: Display help
+[source,terminal]
+----
+$ tkn hub -h
+----
+
+.Example: Interact with hub API server
+[source,terminal]
+----
+$ tkn hub --api-server https://api.hub.tekton.dev
+----
+
+[NOTE]
+====
+For each command examples given below, you can get the corresponding sub-commands and flags by executing `tkn hub <command> --help`.
+====
+
+== hub downgrade
+Downgrade an installed resource
+
+.Example: Downgrade an installed resource
+[source,terminal]
+----
+$ tkn hub downgrade task \
+> --context $(kubectl config current-context) \
+> --kubeconfig $HOME/.kube/config \
+> --namespace $KUBECONFIG \
```

Comment:
> --namespace $KUBECONFIG

should we change the var to `$NAMESPACE` ?

Reply:
Sure. Should I change this for all examples?
BTW, I provided the default values that are available in the tekton CLI:
```
bash-4.4 ~ $ tkn hub downgrade --help
Downgrade an installed resource

Usage:
tkn hub downgrade [flags]
tkn hub downgrade [command]

Available Commands:
  task        Downgrade an installed Task by its name to a lower version

Flags:
  -c, --context string      Name of the kubeconfig context to use (default: kubectl config current-context)
  -h, --help                help for downgrade
  -k, --kubeconfig string   Kubectl config file (default: $HOME/.kube/config)
  -n, --namespace string    Namespace to use (default: from $KUBECONFIG)
      --to string           Version of Resource

Global Flags:
      --api-server string   Hub API Server URL (default "https://api.hub.tekton.dev")

Use "tkn hub downgrade [command] --help" for more information about a command.
``` 
If `$KUBECONFIG` is not the correct default (for both OpenShift and Kubernetes), I request folks with upstream access to make this change for tekton.

Reply:
> > --namespace $KUBECONFIG
> 
> should we change the var to `$NAMESPACE` ?

@sm43 I have changed it to `$NAMESPACE`.

Reply:
if user doesn't pass the flag it will take the namespace value from the kubeconfig file.
hence `from $KUBECONFIG` in the flag description

Reply:
how about `-namespace mynamespace` ? for other `tkn` command we are using `mynamespace`
https://docs.openshift.com/container-platform/4.7/cli_reference/tkn_cli/op-tkn-reference.html#pipeline

Reply:
what if we skip showing the `--context` and `--kubeconfig` flags for hub commands in examples? they are available for almost all tkn command but are not shown in examples and not needed to be passed always

Reply:
@VeereshAradhya - Made the changes you suggested, and some other modifications to make the document look like the other `tkn` command documentation pages.

@sm43 - Removed the `--context` and `--kubeconfig` flags and shortened the commands to a single line.